ENIGMAsystem/SHELL/Universal_System/Extensions/ProcInfo/procinfo.h

```diff
@@ -0,0 +1,142 @@
+﻿/*
+
+ MIT License
+ 
+ Copyright © 2020 Samuel Venable
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ 
+*/
+
+// individual platforms need their platform-specific
+// window types casted to an unsigned long long type
+#define window_t unsigned long long
+
+#ifdef _WiN32
+#define byte __windows_byte_workaround
+#include <windows.h>

```

Comment:
Are you including this because of `process_t`? If so, you should probably find a way around this because it pollutes the user namespace. I cleaned a bunch of this stuff up in #1445 for that reason. I mean, it won't pollute unless your extension is enabled, but regardless.